README.md

```diff
@@ -9,7 +9,7 @@ Specifically, the sidecar monitors the lifecycle of the CRDs generated by the [C
 
 and makes gRPC calls to the associated provisioner.
 
-In this repo you will also find a [sample provisioner](./cmd/sample-driver) based on Minio.
+You can find a [sample provisioner](https://github.com/kubernetes-sigs/cosi-driver-minio) based on Minio.
```

Comment:
Should use https://github.com/kubernetes-sigs/cosi-driver-sample instead as cosi-driver-minio is archived.
Also since COSI is going thru API reviews, it's better to wait until things are settled down before updating this.

Reply:
cosi-driver-sample currently returns `Unimplemented` for every grpc service, but https://github.com/kubernetes-sigs/cosi-driver-minio's implementation may out of date. 
we better update doc to use cosi-driver-sample for now because deploy cosi ref to current doc will run-into no `kustomization.yaml` error.